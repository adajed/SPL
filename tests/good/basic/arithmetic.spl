int main()
{
    printInt(1 << 2);
    printInt(1 << 3);
    printInt(1 << 4);
    printInt((1 << 4) | (1 << 2));

    printInt(4 >> 1);

    printInt(5 & 6);

    return 0;
}
