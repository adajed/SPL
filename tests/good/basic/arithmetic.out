4
8
16
20
2
4
